.github/workflows/check_deps.yml

```diff
@@ -1,11 +1,10 @@
 ---
-
 name: Check Dependencies
 
 on:
   workflow_dispatch:
   schedule:
-    - cron: '38 8 * * *'
+    - cron: '0 8 * * 0'
```

Comment:
How about?
```suggestion
    - cron: '0 8 * * 1'
```

Reply:
Copied that from our other repos, they all have 08**0 🤔